client/src/featureform/resources.py

```diff
@@ -149,21 +149,16 @@ def __init__(self,
                  credentials_path: str,):
 
         self.project_id = project_id
-        self.credentials = self._read_and_serialize_file(credentials_path)
+        self.credentials = json.load(open(credentials_path))
 
     def type(self):
         return "GCPCredentials"
 
     def config(self):
         return {
             "ProjectId": self.project_id,
-            "Credentials": self.credentials,
+            "SerializedFile": self.credentials,
```

Comment:
Remember to change this too

Reply:
Changed it but not sure why it isn't showing here. 

https://github.com/featureform/featureform/blob/bugfix/gcs_filestore/client/src/featureform/resources.py#L160